packages/dd-trace/src/opentracing/span.js

```diff
@@ -240,6 +240,7 @@ class DatadogSpan {
     }
 
     getIntegrationCounter('spans_finished', this._integrationName).inc()
+    this._spanContext._tags['_dd.integration'] = this._integrationName
```

Comment:
Can we move this call up to the constructor?

Reply:
This should be done at the very end when the span finishes.

Reply:
This is fine. The reason I wanted it set in the constructor was to ensure instrumentation telemetry metrics have the correct name, but I see we are already using the integration name in the metrics.